components/src/dynamo/vllm/multimodal_utils/chat_processor.py

```
@@ -0,0 +1,319 @@
+                request,
+                result_generator,
+                request_id,
+                request.model,
+                conversation,
+                self.tokenizer,
+                request_metadata,
+                enable_force_include_usage=False,
+            ):
+                if raw_response.startswith("data: [DONE]"):
+                    break
+                response = json.loads(raw_response.lstrip("data: "))
```

Ruff (0.14.1)

Using `.strip()` with multi-character strings is misleading

(B005)